manifests/r/RescueTime/DesktopApp/2.16.8.2/RescueTime.DesktopApp.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RescueTime.DesktopApp
+PackageVersion: 2.16.8.2
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://www.rescuetime.com/installers/RescueTimeInstaller.exe
+  InstallerSha256: A6CFC215218B562CBE4FD37178D03C2F811E230C4C01AB1B0D6B163323117455
+  ReleaseDate: 2021/11/01

```

All other manifests use the dashed format
```suggestion
  ReleaseDate: 2021-11-01
```